src/slider.jsx

```diff
@@ -469,11 +474,13 @@ const Slider = React.createClass({
       active: false,
     });
     if (this.props.onDragStop) this.props.onDragStop(e);
+    e.preventDefault();
   },
 
   _onDragUpdate(e, pos) {
     if (!this.state.dragging) return;
     if (!this.props.disabled) this._dragX(e, pos);
+    e.preventDefault();
```

Comment:
Please refer to https://github.com/callemall/material-ui/pull/2827 for why it's bad.